node/src/Transaction.ts

```diff
@@ -1613,6 +1614,28 @@ export class BaseTransaction<T extends BaseTransaction<T>> {
         );
     }
 
+    /**
+     * Returns the number of members in the sorted set stored at <code>key</code> with scores between
```

Comment:
Using single quotation instead?
```suggestion
     * Returns the number of members in the sorted set stored at `key` with scores between
```